MediaBrowser.Api/BaseApiService.cs

```diff
@@ -172,16 +172,9 @@ protected DtoOptions GetDtoOptions(IAuthorizationContext authContext, object req
 
                 if (!string.IsNullOrWhiteSpace(hasDtoOptions.EnableImageTypes))
                 {
-                    if (string.IsNullOrEmpty(hasDtoOptions.EnableImageTypes))
-                    {
-                        options.ImageTypes = Array.Empty<ImageType>();
-                    }
-                    else
-                    {
-                        options.ImageTypes = hasDtoOptions.EnableImageTypes.Split(new [] { ',' }, StringSplitOptions.RemoveEmptyEntries)
-                                                                            .Select(v => (ImageType)Enum.Parse(typeof(ImageType), v, true))
-                                                                            .ToArray();
-                    }
+                    options.ImageTypes = hasDtoOptions.EnableImageTypes.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
+                                                                        .Select(v => (ImageType)Enum.Parse(typeof(ImageType), v, true))
+                                                                        .ToArray();
```

Comment:
Cannot you do without `ToArray`?

Reply:
I can, but that would just be a longer version ;)

Reply:
Care to explain more?

Reply:
‘Select’ is a Linq extension it returns ‘IEnumerable’ and we need an array.
‘ToArray’ is the easiest way to convert it.